packages/cli/internal/pkg/cli/workflow/manager.go

```diff
@@ -541,13 +541,14 @@ func (m *Manager) runWorkflow() {
 	if m.err != nil {
 		return
 	}
-	log.Debug().Msgf("running workflow at '%s' with language '%s' and version '%s' using attachments '[%s]', workflow parameters '%s' and engine parameters '%s'",
+	log.Debug().Msgf("calling WES client run_workflow with workflow_url: '%s', workflow_type: '%s', workflow_type_version: '%s', workflow_attachment: '[%s]', workflow_params: '%s', workflow_engine_parameters: '%s'",
```

Comment:
nit: line might be too long and would suggest to separate into shorter to be compliant with go linter. https://golangci-lint.run/usage/linters/#lll